includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -528,12 +528,18 @@ public function javascript_params() {
 		$stripe_params            = [
-			'gatewayId'    => self::ID,
-			'title'        => $this->title,
-			'isUPEEnabled' => true,
-			'key'          => $this->publishable_key,
-			'locale'       => WC_Stripe_Helper::convert_wc_locale_to_stripe_locale( get_locale() ),
-			'apiVersion'   => WC_Stripe_API::STRIPE_API_VERSION,
+			'gatewayId'      => self::ID,
+			'title'          => $this->title,
+			'isUPEEnabled'   => true,
+			'key'            => $this->publishable_key,
+			'locale'         => WC_Stripe_Helper::convert_wc_locale_to_stripe_locale( get_locale() ),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'javascript_params'. (undefined)

(StaticAccess)